lib/ImportStatements.js

```diff
@@ -9,13 +9,15 @@ import uniqBy from 'lodash.uniqby';
 import Configuration from './Configuration';
```

Comment:
I'm okay with 100 I guess--that's what we use at work. I've been resisting because I usually work in a terminal split where 80 characters is just right, but perhaps I just need to get with the times...